### ISSUE TITLE
Missing accessibility lable 

### ISSUE THREAD

**Issue by TStArK1233** [albertvaka/bettercounter#29]
![IMG_20230518_144519](https://github.com/albertvaka/bettercounter/assets/133884373/be2bae4f-742f-4acf-b1ed-b501848e91a5)

The buttons circled in white are not labelled. Thank You for improving it 😊😊.

**Comment by albertvaka** [comment]
Thanks for reporting this! It will be fixed in the next release.
